Build-Wheels.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import requests
+from packaging.requirements import InvalidRequirement
+from packaging.requirements import Requirement
+
+# Global variables for branches and constraints files for the branches to be downloaded
+# (length of each list should be the same and so the positions for branch and it's specific constraints file)
+IDF_BRANCH = ['release/v5.0', 'release/v5.1', 'master']
```

Comment:
Master is already v5.3, so v5.2 is missing.

Can't we use some kind of version detection? It will frequently happen that master becomes newer than the number of the second list indicates. 

Also in cases when the list is constant, it is better to use tuple which is non-mutable and makes clear to the reader that the list won't be changed.

Reply:
Thank you @dobairoland, this is true but there is not v5.3 release on `Github` yet.
So this brings me to think if this whole `idf-python-wheels` shouldn't have to be on `Gitlab` as well to already have build wheels when there is a `Github` release?


Actually I have been already thinking about this suggestion before you have made it and I have tried to make this fully automated now.

Reply:
It is better to handle these builds in one place instead of two. 

I think the main motivation behind having this here is the availability of Windows runners. For the self hosted runners I guess the internal repository would be better.

So IMO we should leave this project on Github, unless @tomassebestik thinks otherwise.